src/keys.c

```diff
@@ -1065,6 +1065,44 @@ int SetCipherSpecs(WOLFSSL* ssl)
         break;
 #endif
 
+#if defined(WOLFSSL_TLS13) && defined(HAVE_NULL_CIPHER)
+    #ifdef BUILD_TLS_SHA256_SHA256
+    case TLS_SHA256_SHA256 :
+        ssl->specs.bulk_cipher_algorithm = wolfssl_cipher_null;
+        ssl->specs.cipher_type           = aead;
```

Comment:
aead is the right choice here?

Reply:
Yes, the draft says to use AEAD and TLS 1.3 only supports AEAD at this time.